# Message to determine the status of the Robot and Penassembly
bool idle
bool busy 
bool error 